metadata.json

```diff
@@ -15,6 +15,10 @@
     "alias"
   ],
   "dependencies": [
+     {
```

Comment:
Oh yeah, I forgot about that. I was just thinking if users didn't have the catalyst set. Adding a dep to simplib means it will have to be on literally every machine using any simp module.